api-reference/beta/api/applicationsecurityprofile_update.md

```diff
@@ -0,0 +1,143 @@
+# Update applicationSecurityProfile
+
+Update the user-editable properties of a applicationSecurityProfile object. **This method must include the applicationSecurityProfileId as a parameter.**
+

```

Comment:
Bold formatting isn't the best way to apply emphasis to the text. Per my previous comment, does this need to be stated, with emphasis? If it needs to be stated, please remove the bold, and state in only one place rather than both the summary and HTTP request sections. It might be clear enough from the syntax/examples. If you want to keep the text, revise so as not to anthropomorphize the method and to focus on the user action: "You must include the application security profile ID as a parameter with this method."